server/bin/test_data.json

```diff
@@ -1983,6 +1983,7 @@
             "name": "Development SKU Product",
             "id": "dev-sku-product",
             "type": "MKT",
+            "skip_pools": "true",
```

Comment:
Nope. It was put in for the derived pool (VDC) stuff. It just creates the Product and skips sub/pool creation. This puts the test data in a state that is more like the CDK setup.